### ISSUE TITLE
pg10 not support 

### ISSUE THREAD

**Issue by longtian001** [timescale/timescaledb#111]
pg10 not support

**Comment by RobAtticus** [comment]
Yes, we do not yet support Postgres 10, but as its release date gets nearer we'll certainly add it to our roadmap.

**Comment by GrayTShirt** [comment]
Release candidate 1 is out. Stable pg10 is around the corner. Is it time to revisit? :)

**Comment by simPod** [comment]
Out already :)

**Comment by mfreed** [comment]
We're working on this!

**Comment by njern** [comment]
Any updates on the timeline for this?

**Comment by RobAtticus** [comment]
@njern Initial support has landed on master, so we welcome volunteers to play around with it and see if there are any issues. This support will also be in 0.7.0, but a few of our release channels will still be based on 9.6 (Ubuntu and RPM most likely).

**Comment by njern** [comment]
Nice, will take `master` for a spin - thanks for the quick answer!

**Comment by simPod** [comment]
PG10 with timescale is working for me so far.

**Comment by RobAtticus** [comment]
Going to close this thread, please feel free to file individual bug reports if you come across something that doesn't work in PG10

**Comment by vjpr** [comment]
@RobAtticus Is PG10 support in `master`?

How to make it install into `/usr/share/postgresql/10`?

What is the env var to specify the version, or how does it detect?

```
/app/timescaledb/build$ sudo make install
[  2%] Built target sqlfile
[  4%] Built target sqlupdatefile
[100%] Built target timescaledb
Install the project...
-- Install configuration: "Release"
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb.control
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.8.0-dev.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.7.1--0.8.0-dev.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.1.0--0.2.0.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.2.0--0.3.0.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.3.0--0.4.0.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.4.0--0.4.1.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.4.1--0.4.2.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.4.2--0.5.0.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.5.0--0.6.0.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.6.0--0.6.1.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.6.1--0.7.0.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.6.1--0.7.1.sql
-- Up-to-date: /usr/share/postgresql/9.6/extension/timescaledb--0.7.0--0.7.1.sql
-- Up-to-date: /usr/lib/postgresql/9.6/lib/timescaledb.so
```

**Comment by RobAtticus** [comment]
@vjpr our build system relies on `pg_config` to tell us where the PostgreSQL installation lives. If the `pg_config` in your `PATH` is 9.6 (you can check with `pg_config --version`) then it will install for that.

You may have to prepend the value of the path to PG10 and it's binaries to your `PATH` in order to override this behavior.

**Comment by joethong** [comment]
hi, does timescale support postgresql 10 now?

**Comment by RobAtticus** [comment]
Yes it does.

**Comment by vjpr** [comment]
@joethong

From http://docs.timescale.com/latest/getting-started/installation/linux/installation-apt

> Note: PostgreSQL 9.6 is required for TimescaleDB on Ubuntu distros. Releases with PostgreSQL 10 are coming soon

**Comment by mfreed** [comment]
TimescaleDB on PG10 is now available also via `apt` using the following command:
```
sudo apt-get install timescaledb-postgresql-10
```
We will be updating docs with the next release to reflect these new naming conventions we'll be using going forward.

**Comment by jbkoh** [comment]
I ran ``sudo apt-get install timescaledb-postgresql-10`` on Ubuntu 16.04, which shows
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 timescaledb-postgresql-10 : Depends: postgresql-10 but it is not installable
                             Depends: postgresql-server-dev-10 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by RobAtticus** [comment]
Hi @jbkoh 

Unfortunately PG10 is not available in Ubuntu's default repositories for 16.04, so you have to follow the instructions at the following link to include Postgresql's repos with have PG10:

http://yallalabs.com/linux/how-to-install-and-use-postgresql-10-on-ubuntu-16-04/

**Comment by jbkoh** [comment]
Thanks it works!
